framework/.eslintrc.json

```diff
@@ -3,11 +3,11 @@
 	"rules": {
 		"camelcase": "off",
 		"max-len": "off",
-		"no-param-reassign": "off",
```

Comment:
Could you explain why do you change this? Also you don't need to wrap it into an array. https://eslint.org/docs/user-guide/configuring#configuring-rules

Reply:
aligning with other packages, but also it has valid reasoning
https://eslint.org/docs/rules/no-param-reassign#disallow-reassignment-of-function-parameters-no-param-reassign